### PR TITLE
add contactpage, user interaction navbar and appear animation for pages

### PR DESCRIPTION
**Pull request by KNT2302**
### Description

- Add contact page
- Add user interaction navbar (animation hovering)
- Add appear animation for each page

### Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Update feature

### Tickets url
[[p - 2], [p - 3], [p - 4]](https://thin-brother-635.notion.site/Add-contact-page-user-interaction-navbar-appear-animation-page-bf1a9eabc46746a6805548e79eb34a00)

### CODE REVIEW

File: src/components/AppHeader.js [hunk 1]
```diff
@@ -1,38 +1,49 @@
...
+      display: "About",
+      to: "/about",
+    },
+    {
+      display: "Portfolio",
+      to: "",
+    },
+    {
+      display: "Contact",
+      to: "/contact",
```
Comment (phuc-create) [inline src/components/AppHeader.js:22]: do you think `label` better than `display`

File: src/style/animation.scss [hunk 2]
```diff
@@ -24,3 +24,42 @@
...
+}
+
+@keyframes page {
+  0%{
+    transform: translateY(15px);
+  }
+  100%{
+    transform: translateY(0px);
+  }
+}
```
Comment (phuc-create) [inline src/style/animation.scss]: new line for this

**Review by phuc-create — CHANGES_REQUESTED** [review]

File: src/components/AppHeader.js (hunk 1 above)
Comment (KNT2302) [reply src/components/AppHeader.js:22]: label is better

Comment (KNT2302) [reply src/components/AppHeader.js:22]: fixed

File: src/style/animation.scss (hunk 2 above)
Comment (KNT2302) [reply src/style/animation.scss]: fixed

**Pull request merged**